Project.toml

```diff
@@ -4,6 +4,7 @@ version = "0.3.6"
 
 [deps]
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
+AdaptiveRejectionSampling = "c75e803d-635f-53bd-ab7d-544e482d8c75"
```

Comment:
I am not sure about this dependency, but don't want to be blocked by this at this stage. We can revisit this later.